draft-packages/button/docs/styles.module.scss

```diff
@@ -0,0 +1,22 @@
+@import "~@kaizen/component-library/styles/responsive";
+
+.buttonSection {
+  display: flex;
```

Comment:
Did you consider using CSS grid for these?

It feels like a perfect opportunity to play with grid and learn how it works (given you're building a grid 😄)

Flex does do the job fine of course, no need to change it unless you want to

Reply:
Will defs use it for the new future stories component